src/components/form/DropdownInput/DropdownInput.js

```diff
@@ -0,0 +1,7 @@
+import { StyledDropdownInput } from "./DropdownInput.styled";
+import { useInputState } from "../../../services/hooks/useInputState";
+
+export const DropdownInput = ({ options, name, label }) => {
```

Comment:
We don't have to write our own implementation of dropdown input.

You can use dropdown from e.g.: semantic ui, and just wrap this dropdown with our styles